### DISCUSSION TITLE
The type or namespace name 'DryIoc' does not exist in the namespace

### DISCUSSION THREAD

**Discussion by sigmarsson** [PrismLibrary/Prism#2861] [Uno Platform]
How can I resolve this error ? 

<img width="695" alt="image" src="https://user-images.githubusercontent.com/43340959/235308740-dcf3a90f-eb52-4a17-ac02-3052401a0faf.png">

I compiled Prism and checked if that type is existing. It is.

**Comment by dansiegel** [comment]
I have no idea you haven't provided any context or sample that reproduces the issue.

**Comment by agtalex** [comment]
@dansiegel I had the same error. For context, I had a class library named `Company.Prism` with derived classes from Prism. Like `public abstract class CommandBase : DelegateCommandBase`. When that library was referenced in the Prism application project I had the following error: `The type or namespace name 'DryIoc' does not exist in the namespace Company.Prism` error came from the `app.xaml.cs.g` of the prism application project.

To workaround I changed the default namespace of the project to `Company.PrismEx` and it works correctly now.

**Reply by dansiegel** [reply]
Yeah you can't do that in general with .NET. The same thing can happen to you if you do something like `MyLibrary.Microsoft`... you'll have huge headaches with most of their packages... and god help you if you did `MyLibrary.System`

**Reply by agtalex** [reply]
Thanks for the quick reply. Looks like it was the same problem in the ticket above. `Weather.History.Prism`.
